templates/automotive/index.html

```diff
@@ -323,236 +403,332 @@ <h2 class="p-heading--5">Over-the-air updates</h2>
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/b061c401-White+paper.svg",
+            alt="whitepaper",
+            width="171",
+            height="150",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img"},
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">whitepaper</h4>
+        </div>
+      </div>
+      <p>Software-defined vehicles: what's the future?</p>
+      <p>Learn how open-source automotive options can boost the industry's shift towards SDV.</p>
+      <p><a href="/engage/software-defined-vehicles-whitepaper">Read here&nbsp;&rsaquo;</a></p>
     </div>
-    <div class="col-6">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/7de55930-canonical-logo1.png",
-        alt="",
-        height="100",
-        width="223",
-        hi_def=True,
-        ) | safe
-      }}
-      <h2 class="p-heading--5">Domain experts</h2>
-      <p class="p-heading--2">Innovate in automotive with Canonical</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">The experts in open source operations, from cloud to desktop to device. The publishers of Ubuntu.</li>
-        <li class="p-list__item is-ticked">Get to market faster.</li>
-        <li class="p-list__item is-ticked">Drive down infra costs.</li>
-        <li class="p-list__item is-ticked">Accelerate innovation.</li>
-        <p>
-          <a class="js-invoke-modal" href="/internet-of-things/contact-us?product=automotive">
-            Get in touch&nbsp;&rsaquo;
-          </a>
-        </p>
+    <div class="col-3 p-divider__block">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/3769b4a7-webinar-icon.svg",
+            alt="",
+            width="88",
+            height="72",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img"},
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Videos</h4>
+        </div>
       </div>
+      <p>Watch Ubuntu Masters at Scania discuss HPC clusters deployment</p>
+      <p><a href="https://www.brighttalk.com/webcast/6793/416900/mastering-multicloud-with-slurm-and-juju">Watch the video&nbsp;&rsaquo;</a></p>
     </div>
-  </section>
+    <div class="col-3 p-divider__block">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg",
+            alt="",
+            width="32",
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img"},
+            ) | safe
+          }}
+          <h4 class="p-heading-icon__title">Announcement</h4>
+        </div>
+      </div>
+      <p>Stay tuned!</p>
```

Comment:
You might want to get another opinion on this, but to me this feels a little off-brand.

Reply:
I will just remove this column then!